big_tests/tests/gdpr_SUITE.erl

```diff
@@ -544,6 +549,34 @@ retrieve_inbox(Config) ->
               Bob, Config, "inbox", ExpectedHeader, ExpectedBobItems)
         end).
 
+remove_inbox(Config) ->
+    escalus:fresh_story(Config, [{alice, 1}, {bob, 1}], fun(Alice, Bob) ->
+            BobU = escalus_utils:jid_to_lower(escalus_client:username(Bob)),
+            BobS = escalus_utils:jid_to_lower(escalus_client:server(Bob)),
+            AliceU = escalus_utils:jid_to_lower(escalus_client:username(Alice)),
+            AliceS = escalus_utils:jid_to_lower(escalus_client:server(Alice)),
+            Body = <<"With spam?">>,
+            send_and_assert_is_chat_message(Bob, Alice, Body),
+
+            ExpectedHeader = ["jid", "content", "unread_count", "timestamp"],
+            maybe_stop_and_unload_module(mod_inbox, mod_inbox_backend, Config),
+            {0, _} = unregister(Alice, Config),
+            {0, _} = unregister(Bob, Config),
```

Comment:
Actually, more interesting would be the test, in which we do not register the second user, and we validate that he still has an inbox after removal of the first one

Reply:
You mean _do not unregister_ ?

Reply:
Yes, sorry, this is what I meant.
Just `unregister` Alice

Reply:
but still, validate that Alice has an empty inbox